vizro-core/examples/visual-vocabulary/assets/css/custom.css

```diff
@@ -11,21 +11,29 @@ img[src*="#chart-icon"] {
 }
 
 .code-clipboard {
+  float: right;
   font-size: 20px;
-  position: absolute;
-  right: 14px;
-  top: 12px;
+}
+
+.code-clipboard pre {
+  font-family: monospace;
+  margin-bottom: 12px;
 }
 
 .code-clipboard-container {
   background: var(--surfaces-bg-card);
-  font-family: monospace;
   max-height: 500px;
   overflow: auto;
   padding: 1rem;
   position: relative;
 }
 
+.code-clipboard-container .pycafe-link,
```

Comment:
To be discussed with @huong-li-nguyen: should we do this styling for other buttons. e.g. the one on the 404 page looks a bit weird I think.

Reply:
Also, did we consider making the `code-clipboard-container` always stretched to the bottom of the page so it's always aligned with the bottom of the chart?

Reply:
I think @huong-li-nguyen spent a while playing around with adjusting the size and stretching the code clipboard container and decided the current solution is best, but let's check when she gets back.

What else do you think could be improved in terms of UI?

Reply:
I can’t say for sure, but here are a few ideas we might consider:

1. Moving the button between the tab selection and clipboard icon, placing it logically outside the "copy-button" area.
2. Adding a small PyCafe icon within the button text (and possibly shortening the text if it improves clarity).
3. Including a link icon 🔗 (or a similar symbol) to indicate that clicking the button will open a new browser tab.

These are minor suggestions, so feel free to disregard if they don’t quite fit! 😄

Reply:
All good points I think! I will unresolve this conversation after merging so we can revisit when @huong-li-nguyen is back. My ideal UI here would actually probably not be tabs at all but instead a toggle switch where it highlights/unhighlights the Vizro code as you toggle on/off. The current solution is really just meant as a relatively quick and easy addition to the UI that works well enough for now rather than a completely polished one.

1. I tried this and found it looked worse rather than better 😅 
2. I did play around with a pycafe icon on the button for a while and couldn't get anything that worked well with our colors. I also tried using the coffee emoji which looks pretty similar but didn't like it much. 
3. I've put in an "opens in new tab" icon and it looks good I think!
![image](https://github.com/user-attachments/assets/23da03b9-67bd-4e32-a6b3-eda55b36f8f5)